src/riverroute/mosart_restfile.F90

```diff
@@ -0,0 +1,455 @@
+module mosart_restfile
+
+   ! Read from and write to the MOSART restart file.
+
+   use shr_kind_mod,       only : r8 => shr_kind_r8, CL => shr_kind_cl
+   use shr_sys_mod,        only : shr_sys_abort
+   use mosart_vars,        only : iulog, inst_suffix, caseid, nsrest, &
+                                  spval, mainproc, nsrContinue, nsrBranch, nsrStartup, &
+                                  ctitle, version, username, hostname, conventions, source
+   use mosart_data,        only : ctl, Trunoff
+   use mosart_histfile,    only : mosart_hist_restart
+   use mosart_fileutils,   only : getfil
+   use mosart_timemanager, only : timemgr_restart, get_nstep, get_curr_date
+   use mosart_io
```

Comment:
Add only clause

Reply:
Same as https://github.com/ESCOMP/MOSART/pull/76#discussion_r1629728441

Reply:
This has been resolved.